docs/guides/admin/docs/configuration/basic.md

```diff
@@ -3,106 +3,102 @@ Basic Configuration
-set to the same value on each of your Opencast nodes (Core, Worker, Capture Agent, …)
+Make sure that these settings are identical on all nodes of the cluster.
 
 
-Step 3: Change the default shutdown command
--------------------------------------------
+Step 3: Setting up Apache ActiveMQ Message Broker
+-------------------------------------------------
 
-Karaf provides a socket over which you can send a shutdown command. The socket does not provide any kind of
-authentication. Therefore anyone who obtains write access to this socket is able to shutdown karaf and everything
-that runs on it. There is a default `karaf.shutdown.command` defined in `custom.properties`. Change this to something
-secret.
+Opencast requires Apache ActiveMQ to relay messages between micro-services.
+For configuration details, please follow the:
 
+- [Apache ActiveMQ configuration guide](message-broker.md)
 
-Step 4: Setting up Apache ActiveMQ Message Broker
--------------------------------------------------
-<!-- _This is a very nonchalant and subdued description for how important this config probably is -->
-Since version 2.0, Opencast requires a running Apache ActiveMQ instance with a specific configuration.  The message
-broker is mostly run on the admin server of Opencast but can be run separately. It needs to be started before Opencast.
-For more details about the setup, have a look at the [Apache ActiveMQ configuration guide](message-broker.md).
 
-
-Step 5: Database Configuration
+Step 4: Database Configuration
 ------------------------------
 
-Opencast uses an integrated H2 database by default. While you will find it perfectly functional, it has certain
-drawbacks:
+Opencast uses an integrated H2 database by default, which has certain drawbacks:
 
-* It is rather slow
 * It cannot be used for distributed set-ups
 * Upgrading Opencast with this database is not possible
 
-For testing, it is totally fine to keep the internal database, but you are highly encouraged to switch to a stand-alone
-database for productional use. For more information about database configuration, have a look at the [Database
-Configuration](database.md) section.
+For testing, it is fine to keep the internal database, but configure a stand-alone database for production.
+Details about the configuration can be found at:
+
+- [Database Configuration](database.md)
+
 
-Step 6: Elasticsearch Configuration
+Step 5: Elasticsearch Configuration
 -----------------------------------
 
-Opencast requires Elasticsearch.
-For more details about the setup, have a look at the [Elasticsearch configuration
-guide](../modules/searchindex/elasticsearch.md).
+Opencast requires Elasticsearch for quick full-text search and data caching.
+For more details about the setup, have a look at the:
 
-Step 7: HTTPS Configuration
+- [Elasticsearch configuration guide](../modules/searchindex/elasticsearch.md).
+
+
+Step 6: HTTPS Configuration
```

Comment:
```suggestion
Step 6: HTTPS Configuration (Add external access)
```

Reply:
Let's name it “Reverse Proxy Configuration”. That includes both :D